Makefile

```diff
@@ -12,7 +12,11 @@ errcheck:
 fmt:
 	@if [ -n "$$(go fmt ./...)" ]; then echo 'Please run go fmt on your code.' && exit 1; fi
 
-install_dependencies: install_errcheck get
+install_dependencies: install_errcheck glide_install
+
+glide_install:
+	go get -u github.com/Masterminds/glide
+	cd $(TRAVIS_BUILD_DIR) && glide install --strip-vendor
```

Comment:
Can't we use something like `$(GOPATH)/src/github.com/remerge/sarama` instead of using the Travis env var here?

Reply:
I used the travis build dir so it runs with my fork. As travis is now enabled there is no real need to run it on a fork 😸